src/DeviceLogSorted.cpp

```diff
@@ -39,7 +45,7 @@ cDeviceLogSorted::cDeviceLogSorted()
 
 void cDeviceLogSorted::InitDeviceForWriting(int pHandle, std::string timestamp, std::string directory, uint64_t maxDiskSpace, uint32_t maxFileSize)
 {
-    for (uint32_t i = 0; i < DID_COUNT; i++)
+	for (uint32_t i = 0; i < DID_COUNT; i++)
```

Comment:
Tabs or Spaces?  Looks like we have some mixed spacing going on?

Reply:
Do you have a good way of preventing this?  I think vscode is supposed to detect the indentation style and use the similar one.  I can spend time arrow keying through the start of each line to determine how it's being indented.